docs/advanced_usage.rst

```diff
@@ -23,6 +23,31 @@ You can also use a Unix Domain Socket to connect to the agent::
     tracer.configure(uds_path="/path/to/socket")
 
 
+.. _disabling_tracing:
+
+Disabling Tracing
```

Comment:
TODO: Add `ddtrace.writer` module to API docs?

My thinking was maybe not just yet? they are public, but maybe try to get some usage out of the module first before we document in an encouraging way?